operating/manage-compliance/managing-compliance-20.adoc

```diff
@@ -0,0 +1,27 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="managing-compliance-20"]
+= Managing compliance (2.0)
+include::modules/common-attributes.adoc[]
+:context: managing-compliance-20
+
+toc::[]
+
+:FeatureName: Compliance (2.0)
+include::snippets/technology-preview.adoc[]
+
+By navigating to the Compliance (2.0) section in the {rh-rhacs-first} portal, you can view the compliance results associated with your integration by using the compliance operator.
+
+//Viewing the cluster compliance page
+include::modules/viewing-the-cluster-compliance-page.adoc[leveloffset=+1]
+
+//Understanding the cluster compliance page
+include::modules/understanding-the-cluster-compliance-page.adoc[leveloffset=+1]
+
+//Creating a compliance scan schedule
+include::modules/creating-a-compliance-scan-schedule.adoc[leveloffset=+1]
+
+//Viewing the compliance scan status
+include::modules/viewing-the-compliance-scan-status.adoc[leveloffset=+1]
```

Comment:
In addition to viewing results across clusters, will there be documentation on what happens when scan configurations are deleted?

I ask because any scan results associated with a scan configuration will be removed from the database, and the secured clusters. Will we want to call out the importance of backing up compliance data before deleting a scan configuration?

Reply:
@rhmdnd Thank you for your suggestion. However, I'd like @dashrews78 to chime in on this since I added documentation based on the feature demo David and Brad created: https://docs.google.com/document/d/1tXoTYgq6R1rIVe2qTSdw4-sM6kFBCjtHTQarfW45f3Y/edit#bookmark=id.ceui8sfk19zf

Reply:
@rhmdnd Does such documentation exist in CO docs?  The same concept exists if working directly with CO, ACS is just making it easier.  I'm wondering if a popup when trying to delete a scan config is sufficient instead of documentation.

Reply:
Yeah - a popup is probably fine for now.

Perhaps @sheriff-rh and I can work on a more complete write-up in the Compliance Operator documentation that describes how a user backs up all their compliance data.

This is probably good practice anyway, since it outputs compliance documents/evidence that some folks (auditors) might not have access to without direct access to the cluster.

Reply:
yeah but we would want to be careful of how much of that leaks into ACS docs as ACS would want to do such a thing via the functionality resulting from https://issues.redhat.com/browse/ROX-22365

Reply:
That makes sense. A generic popup is probably fine for now.

If/when we document backup procedures in the compliance operator documentation, we can leave that out of the ACS documentation, or supplement it with ACS functionality.